resources/views/movies/show.blade.php

```diff
@@ -97,20 +99,27 @@
                 </div>
                 <div class="w-full lg:pr-24 md:pr-16 flex flex-col md:items-start md:text-left mb-16 md:mb-0 items-center text-center">
                     <h1 class="font-medium text-gray-500 text-4xl py-16">Recent Reviews</h1>
-                    @foreach($movie->reviews as $review)
+                    @foreach($reviews as $review)
                         <div class="flex-col py-6">
-                            <h1 class="font-bold text-lg tracking-wide"> <svg class="w-5 h-5 inline-flex align-top" xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" stroke="currentColor">
-                                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M11.049 2.927c.3-.921 1.603-.921 1.902 0l1.519 4.674a1 1 0 00.95.69h4.915c.969 0 1.371 1.24.588 1.81l-3.976 2.888a1 1 0 00-.363 1.118l1.518 4.674c.3.922-.755 1.688-1.538 1.118l-3.976-2.888a1 1 0 00-1.176 0l-3.976 2.888c-.783.57-1.838-.197-1.538-1.118l1.518-4.674a1 1 0 00-.363-1.118l-3.976-2.888c-.784-.57-.38-1.81.588-1.81h4.914a1 1 0 00.951-.69l1.519-4.674z" />
-                                </svg> {{ $review->rating }} <span class="font-normal text-sm">/ 10</span> </h1>
-                            <h2 class="font-medium pt-2 text-md"> {{ $review->title }} </h2>
+                            <a href="{{ route('reviews.show', ['movie' => $movie, 'review' => $review]) }}">
+                                <h1 class="font-bold text-lg tracking-wide"> <svg class="w-5 h-5 inline-flex align-top" xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24" stroke="currentColor">
+                                        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M11.049 2.927c.3-.921 1.603-.921 1.902 0l1.519 4.674a1 1 0 00.95.69h4.915c.969 0 1.371 1.24.588 1.81l-3.976 2.888a1 1 0 00-.363 1.118l1.518 4.674c.3.922-.755 1.688-1.538 1.118l-3.976-2.888a1 1 0 00-1.176 0l-3.976 2.888c-.783.57-1.838-.197-1.538-1.118l1.518-4.674a1 1 0 00-.363-1.118l-3.976-2.888c-.784-.57-.38-1.81.588-1.81h4.914a1 1 0 00.951-.69l1.519-4.674z" />
+                                    </svg> {{ $review->rating }} <span class="font-normal text-sm">/ 10</span> </h1>
+                                <h2 class="font-medium pt-2 text-md"> {{ $review->title }} </h2>
+                            </a>
                             <span class="font-bold text-sm text-blue-500 mr-4"> {{ $review->user->name }} </span>
                             <span class="font-normal text-sm"> {{ $review->created_at->diffForHumans() }} </span>
-                            <p class="py-2"> {{ $review->content }} </p>
+                            <p class="py-2">
+                                {{ \Illuminate\Support\Str::limit($review->content, 200, '...') }}
```

Comment:
Perhaps this could be handle in the controller? Or maybe model if you use this in other places?
Maybe you can call this $review->excerpt?

Reply:
So similar to what you have done with the movie duration 
You can add a getExcerptAttribute on the Review model :D and return the return in that format 😄